src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -828,7 +828,7 @@ for n,sFile in ipairs( tApis ) do
     end
 end
 
-if turtle then
+if turtle and fs.isDir( "rom/apis/turtle" ) then
```

Comment:
It also makes sense to check for that on off chance that this folder accidentally (on on purpose) was removed in jar file.